util/util.go

```diff
@@ -101,13 +101,20 @@ func WriteFile(fs billy.Basic, filename string, data []byte, perm os.FileMode) e
 	if err != nil {
 		return err
 	}
+	defer func() {
+		if f != nil {
+			_ = f.Close()
```

Comment:
If we name the returning error, we could:

```suggestion
			err = f.Close()
```

And remove lines 115-119.